Source/Core/Core/HW/GBACore.h

```diff
@@ -0,0 +1,106 @@
+// Copyright 2021 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <array>
+#include <condition_variable>
+#include <memory>
+#include <mutex>
+#include <queue>
+#include <thread>
+#include <vector>
+
+#define PYCPARSE  // Remove static functions from the header
+#include <mgba/core/interface.h>
+#undef PYCPARSE
+#include <mgba/core/core.h>
+#include <mgba/gba/interface.h>
+
+#include "Common/CommonTypes.h"
+
+class PointerWrap;
+
+namespace HW::GBA
+{
+static constexpr auto BITS_PER_SECOND = 115200;
```

Comment:
This has been shown to be wrong--see #9552.

Reply:
That was mentioned in this PR, the goal isn't to improve compatibility but address the usability of this feature. Once more accurate timings get merged I will rebase

Reply:
Yes I only noticed that after I posted this, apologies for the noise.